include/zephyr/drivers/interrupt_controller/loapic.h

```diff
@@ -82,7 +82,7 @@ DEVICE_MMIO_TOPLEVEL_DECLARE(loapic_regs);
 static inline uint32_t x86_read_xapic(unsigned int reg)
 {
 	mm_reg_t base;
-    base = DEVICE_MMIO_TOPLEVEL_GET(loapic_regs);
```

Comment:
You added these mistakes in you first patch, so fix your first patch, do not create a fixup patch.

Reply:
Hi @tbursztyka ,
Thanks for point it out. This has been fixed now.